### ISSUE TITLE
Running seq on WSL2

### ISSUE THREAD

**Issue by RenzoTale88** [seq-lang/seq#231]
Hi,
I've installed seq using the command line specified on the website. However, when I run it I come across an Illegal instruction error:
```
~/.seq/bin/seqc --help
Illegal instruction
```

I've seen the issue #229 and I've checked the cpu info, which are as follow:
```
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 30
model name      : Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz
stepping        : 5
microcode       : 0xffffffff
cpu MHz         : 2793.000
cache size      : 8192 KB
physical id     : 0
siblings        : 8
core id         : 0
cpu cores       : 4
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 11
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx rdtscp lm constant_tsc rep_good nopl xtopology cpuid pni ssse3 cx16 sse4_1 sse4_2 popcnt hypervisor lahf_lm pti ssbd ibrs ibpb stibp flush_l1d arch_capabilities
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds swapgs itlb_multihit
bogomips        : 5586.00
clflush size    : 64
cache_alignment : 64
address sizes   : 36 bits physical, 48 bits virtual
power management:
```
The cpu is admittedly old, but it does support sse4.2. Is there any other reason why it should not work?
Thank you in advance for your help,
Andrea

**Comment by arshajii** [comment]
Hi @RenzoTale88 — would it be possible to run `seqc` in gdb to see what instruction causes the issue (e.g. as described [here](https://stackoverflow.com/a/40223712))? Then we can fix it on our end in the next release.

**Comment by arshajii** [comment]
FYI we think we know what's causing this, and should be fixed in the next release in a few days.

**Comment by arshajii** [comment]
This should be fixed in 0.10.3 (released yesterday). Feel free to reopen if the issue persists.
